src/main/java/com/snowflake/snowpark_java/SessionBuilder.java

```diff
@@ -80,4 +80,16 @@ public Session create() {
   public Session getOrCreate() {
     return new Session(this.builder.getOrCreate());
   }
+
+  /**
+   * Adds the app name to set in the query_tag after session creation
```

Comment:
Let's also say in this doc string that this will set the query tag to `APPNAME=${appName}` so they know what to expect in the query tag. 

Also, how do we handle the case if they use .appName() *and* `.setQueryTag()`?

Reply:
Thank you. I will add the recommendation in the doc.
The appName() and setQueryTag can be overwritten by the user at any time. It's not the common pattern but its allowed.